src/test/java/com/synopsys/integration/jira/common/server/IssueServiceTestIT.java

```diff
@@ -82,10 +86,20 @@ public void createIssueWithCustomFieldTest(JiraHttpClient jiraHttpClient) throws
         issueRequestModelFieldsBuilder.setDescription("Test description");
         issueRequestModelFieldsBuilder.setPriority("3");
 
-        String key = "customfield_10107";
+        String key = "customfield_10700";
```

Comment:
Should this be passed in through the environment?

Reply:
It should be possible. We have JiraTestEnvVars currently which we use to pass in Jira Server/Cloud properties. I can add another one for a custom field to be used by tests such as JIRA_CUSTOM_FIELD_TEST_ID